wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -128,6 +127,8 @@ class WireGenerator(
 }
 
 private fun parseFileDescriptor(fileDescriptor: FileDescriptorProto, descs: DescriptorSource): ProtoFileElement {
+  val packageName = if (fileDescriptor.hasPackage()) fileDescriptor.`package` else null
```

Comment:
nit: I think you could inline this below since `packageName` isn't used anywhere except in the `return` line.